e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-            "RMSE\nCORR",
-            ha="left",
-            fontdict=plotSideTitle,
-        )
-        fig.text(
-            panel[n][0] + 0.45,
-            panel[n][1] + 0.0,
-            "%.2f\n%.2f" % stats[3:5],
-            ha="right",
-            fontdict=plotSideTitle,
-        )
-
-
-def plot(reference, test, diff, metrics_dict, parameter):
-    # Create figure, projection
-    fig = plt.figure(figsize=parameter.figsize, dpi=parameter.dpi)
-
-    # Create projection
-    if parameter.var_region.find("N") != -1:
-        pole = "N"
-        proj = ccrs.NorthPolarStereo(central_longitude=0)
-    elif parameter.var_region.find("S") != -1:
-        pole = "S"
-        proj = ccrs.SouthPolarStereo(central_longitude=0)
-
-    # First two panels
-    min1 = metrics_dict["test"]["min"]
-    mean1 = metrics_dict["test"]["mean"]
-    max1 = metrics_dict["test"]["max"]
-    if test.count() > 1:
-        plot_panel(
-            0,
-            fig,
-            proj,
-            pole,
-            test,
-            parameter.contour_levels,
-            parameter.test_colormap,
-            [parameter.test_name_yrs, parameter.test_title, test.units],
-            parameter,
-            stats=(max1, mean1, min1),
-        )
-
-    min2 = metrics_dict["ref"]["min"]
-    mean2 = metrics_dict["ref"]["mean"]
-    max2 = metrics_dict["ref"]["max"]
-
-    if reference.count() > 1:
-        plot_panel(
-            1,
-            fig,
-            proj,
-            pole,
-            reference,
-            parameter.contour_levels,
-            parameter.reference_colormap,
-            [
-                parameter.ref_name_yrs,
-                parameter.reference_title,
-                reference.units,
-            ],
-            parameter,
-            stats=(max2, mean2, min2),
-        )
-
-    # Third panel
-    min3 = metrics_dict["diff"]["min"]
-    mean3 = metrics_dict["diff"]["mean"]
-    max3 = metrics_dict["diff"]["max"]
-    r = metrics_dict["misc"]["rmse"]
-    c = metrics_dict["misc"]["corr"]
-
-    if diff.count() > 1:
-        plot_panel(
-            2,
-            fig,
-            proj,
-            pole,
-            diff,
-            parameter.diff_levels,
-            parameter.diff_colormap,
-            [None, parameter.diff_title, test.units],
-            parameter,
-            stats=(max3, mean3, min3, r, c),
-        )
-
-    # Figure title
-    fig.suptitle(parameter.main_title, x=0.5, y=0.97, fontsize=18)
-
-    # Save figure
-    for f in parameter.output_format:
-        f = f.lower().split(".")[-1]
-        fnm = os.path.join(
-            get_output_dir(parameter.current_set, parameter),
-            parameter.output_file + "." + f,
-        )
-        plt.savefig(fnm)
-        # Get the filename that the user has passed in and display that.
-        fnm = os.path.join(
-            get_output_dir(parameter.current_set, parameter),
-            parameter.output_file + "." + f,
-        )
```

Comment:
This doesn't seem to have a correspondence, but it is identical to the block above. Therefore, it appears to be redundant.